lib/telegram-bot.js

```diff
@@ -20,21 +21,70 @@ function hasActiveChats$() {
   );
 }
 
-function notify(message) {
+
+function cal(m) {
```

Comment:
No abbreviations, please. Both for the function name and the variable name.

Reply:
fixed